Source/MapMgr.cpp

```diff
@@ -158,7 +158,7 @@ namespace GW {
     }
 
     int Map::GetDistrict() {
-        CharContext *ctx = GameContext::instance()->character;
+        WorldContext* ctx = GW::GameContext::instance()->world;
```

Comment:
Why this change? Is `CharContext::district_number` not good anymore ?

Reply:
I noticed that Trade alerts in toolbox were triggering even when in Kama ae1 - this was because `CharContext::district_number` is always 0 now ( I think the devs changed this in relation to the update notes from the other day - https://wiki.guildwars.com/wiki/Game_updates )